wandb/__init__.pyi

```diff
@@ -399,15 +399,15 @@ def log(
     commit: Optional[bool] = None,
     sync: Optional[bool] = None,
 ) -> None:
-    """Log a dictionary of data to the current run's history.
+    """Upload run data.
 
-    Use `wandb.log` to log data from runs, such as scalars, images, video,
+    Use `log` to log data from runs, such as scalars, images, video,
     histograms, plots, and tables.
 
     See our [guides to logging](https://docs.wandb.ai/guides/track/log) for
     live examples, code snippets, best practices, and more.
 
-    The most basic usage is `wandb.log({"train-loss": 0.5, "accuracy": 0.9})`.
+    The most basic usage is `run.log({"train-loss": 0.5, "accuracy": 0.9})`.
```

Comment:
quick question - curious why this change here? Is run a better way to refer to an instance of wandb?

Reply:
I'd like to encourage users to use an explicit `run`:

```
run = wandb.init()
run.log(...)
```

Using `wandb.log()` uses the implicit global run, which is OK for short scripts and demos, but can become hard to maintain as the script grows. Being explicit here is just a good software engineering practice.

Reply:
As well as when we support multiple parallel runs in same script. Good to get users accustomed to this usage pattern